actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -73,6 +70,9 @@ def flush_newline_if_pending(src)
         end
       end
 
+      # For backwards compatibility
+      Erubis = Erubi
```

Comment:
I think we have to do that because we advertise the Erubis class here http://api.rubyonrails.org/classes/ActionView/Template/Handlers/Erubis.html

Then we're in an even better position to default to Erubi as the ERB implementation because people have an out.

I think Active Support Deprecation has a method to deprecate a constant which could help here. 👍